raster/loader/raster2pgsql.c

```diff
@@ -1564,7 +1564,7 @@ convert_raster(int idx, RTLOADERCFG *config, RASTERINFO *info, STRINGBUFFER *til
 	GDALDatasetH hdsSrc;
 	GDALRasterBandH hbandSrc;
 	int nband = 0;
-	int i = 0;
+	uint32_t i = 0;
 	int ntiles[2] = {1, 1};
 	int _tile_size[2] = {0, 0};
```

Comment:
can _tile_size be also converted to unsigned here instead of below?

Reply:
It does some subtractions like:
```
_tile_size[1] = info->dim[1] - (ytile * info->tile_size[1]);
```
Without going too deep into it I'm not able to determine if it'd go under zero.

Reply:
can you do that adding an `assert(info->dim[1] >= (ytile * info->tile_size[1]))`?